src/interface.rs

```diff
@@ -0,0 +1,21 @@
+use anyhow::bail;
+use execution::types::{Interface, Ledger};
+use std::sync::Mutex;
+
+lazy_static::lazy_static! {
+   pub static ref MEM: Mutex::<Ledger> = Mutex::new(Ledger::new());
```

Comment:
It's close to how I make static settings loading in massa with custom configs in test works :)